Travis/js/main.js

```diff
@@ -0,0 +1,154 @@
+// Create an object that has a property that is an array. Log one of the elements of that array.
+	
+	myObj={
+		myObjArray:["one","two","three"]
+	}
+
+	console.log(myObj.myObjArray[2])
+
+// Create an object that has a property that is an object. Log one of the properties of that inner object.
+
+	myObjTwo = { 
+		property1:{
+			otherProp1 : 50,
+			otherProp2	: 60
+		}
+	}
+
+	console.log(myObjTwo.property1.otherProp1)
+
+// Create an object that has a property that is a function (method). Call that method.
+
+	myObjThree = {
+		myFunct:()=>{
+			console.log("MyFunctionIsWorking");
+		}
+
+	}
+
+	myObjThree.myFunct();
+
+// Create an array that has an object in it. Log one of the properties of that object.
+
+	myArray=[
+		
+		myObjFour = {property1: 100, property2: 200}
+	]
+
+	console.log(myArray[0].property1)
+
+// Create an array that has an array as one of its elements. Log one of the elements of the inner array.
+
+	myArrayTwo=[
+		myArrayThree = ["property1", "property2",]
```

Comment:
same comment as above -- dont do an assignment AS an array element, just put the Array in there as a literal.  also somehow you're getting away with an extra comma in there.